site/content/en/references/kustomize/kustomization/replacements/_index.md

```diff
@@ -0,0 +1,258 @@
+---
+title: "replacements"
+linkTitle: "replacements"
+type: docs
+weight: 18
+description: >
+    Substitute field(s) in N target(s) with a field from a source.
+---
+
+Replacements are used to copy fields from one source into any
+number of specified targets. The `replacements` field in the kustomization file can support either a path to 
+a replacement or an inline replacement. 
+
+`kustomization.yaml`
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+replacements:
+  - path: replacement.yaml
+  - source:
+      kind: Deployment
+      fieldPath: metadata.name
+     targets:
+       - select: 
+           name: my-resource
+```
+
+### Syntax
+
+The full schema of `replacements` is as follows:
+
+```yaml
+replacements:
+- source:
+    group: string
+    version: string
+    kind: string
+    name: string
+    namespace: string
+    fieldPath: string
+    options:
+      delimiter: string
+      index: int
+      create: bool
+  targets: 
+  - select:
+      group: string
+      version: string
+      kind: string
+      name: string
+      namespace: string
+    reject:
+    - group: string
+      version: string
+      kind: string
+      name: string
+      namespace: string
+    fieldPaths: 
+    - string
+    options:
+      delimiter: string
+      index: int
+      create: bool
+```
+
+### Field Descriptions
+
+| Field       | Description |
+| ----------- | ----------- |
+| source [required] | The source of the value |
+| target [required] | The N fields to write the value to |
+| group [optional]  | The group of the referent |
+| version [optional] | The version of the referent
+|kind [optional] |The kind of the referent
+|name [optional] |The name of the referent
+|namespace [optional] |The namespace of the referent
+|select [required] |Include objects that match this
+|reject [optional] |Exclude objects that match this
+|fieldPath [optional] |The structured path to the source value (defaults to metadata.name)
+|fieldPaths [optional] |The structured path(s) to the target nodes (defaults to [metadata.name])
+|options [optional] |Options used to refine interpretation of the field
+|delimiter [optional] |Used to split/join the field
+|index [optional] |Which position in the split to consider (defaults to 0)
+|create [optional] |If target field is missing, add it (defaults to false)
+
+### Source and Targets
+The source selector must resolve to a single source. The targets can be ambiguous, and replacements will be applied to all matching targets.
```

Comment:
The `source` field is a selector that ....

Suggest breaking `Source` and `Target` into distinct sections, since Target has both a `select` and `reject`, while `source` is implicitly select only.